packages/backend/src/clients/github/Github.ts

```diff
@@ -238,15 +241,19 @@ export const createFile = async ({
 }) => {
     const octokit = getOctokitRestForApp(installationId);
 
-    const response = await octokit.rest.repos.createOrUpdateFileContents({
-        owner,
-        repo,
-        path: fileName,
-        message,
-        content: Buffer.from(content, 'utf-8').toString('base64'),
-        branch,
-    });
-    return response;
+    try {
```

Comment:
added try catch to some github calls, to avoid the unexpected Server Error  message